test/recipes/80-test_cmp_http.t

```diff
@@ -278,17 +277,18 @@ sub start_mock_server {
     my $pid = open($server_fh, "$cmd|") or die "Trying to $cmd";
     print "Pid is: $pid\n";
     # Find out the actual server port
+    $server_port = "error";
     while (<$server_fh>) {
         print;
         s/\R$//;                # Better chomp
-        next unless (/^ACCEPT\s.*:(\d+)$/);
-        $server_port = $1;
-        $server_tls = $1;
-        $kur_port = $1;
-        $pbm_port = $1;
+        next unless (/^ACCEPT/);
+        $server_port = $server_tls = $kur_port = $pbm_port = $1
+            if m/^ACCEPT\s.*:(\d+)$/;
```

Comment:
May I suggest a small modification?

``` perl
if m/^ACCEPT\s.*?:(\d+)$/;
```

`*?` is the non-greedy variant of `*`, and is sometimes kinder for the regexp machine...